edx_sysadmin/templates/edx_sysadmin/base.html

```diff
@@ -1,8 +1,23 @@
-{% extends "main_django.html" %}
 {% load i18n static %}
```

Comment:
Why are we removing the `{% extends "main_django.html" %}` base template?

Reply:
Ok, I guess, you were pointing this template from platform. let me add it back.